apps/src/templates/projects/ProjectCardGrid.story.jsx

```diff
@@ -73,17 +109,27 @@ export default storybook => {
         description: 'Class gallery sorted by recency of when the project was added.',
         story: () => (
           <ProjectCardGrid
-            projects = {generateFakeClassProjects()}
+            projectLists = {generateFakeClassProjects()}
             galleryType = "class"
           />
         )
       },
       {
-        name: 'Public Gallery',
-        description: 'Public gallery sorted by project type AND recency of when the project was added.',
+        name: 'Public Gallery with student info',
+        description: 'Public gallery sorted by project type AND recency of when the project was added, without student name and age.',
+        story: () => (
+          <ProjectCardGrid
+            projectLists = {generateFakePublicProjectsWithStudentInfo()}
+            galleryType = "public"
+          />
+        )
+      },
+      {
+        name: 'Public Gallery without student info',
+        description: 'Public gallery sorted by project type AND recency of when the project was added, without student name and age.',
```

Comment:
Should the Stories also reflect that the sorting/grouping isn't happening yet? Or is it still applicable since they are sorted/grouped server side?